account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -329,7 +336,235 @@ describe("MA v2 Tests", async () => {
     );
   });
 
-  it("installs a session key via deferred action signed by the owner and has it sign a UO", async () => {
+  it("installs a session key via deferred action using PermissionBuilder signed by the owner and has it sign a UO", async () => {
+    let provider = (
+      await givenConnectedProvider({
+        signer,
+      })
+    )
+      .extend(installValidationActions)
+      .extend(deferralActions);
+
+    await setBalance(client, {
+      address: provider.getAddress(),
+      value: parseEther("2"),
+    });
+
+    const { entityId, nonce } = await provider.getEntityIdAndNonce({
```

Comment:
same here, is this required to be on the client? does it use the client to make requests?